netbox/templates/inc/table_controls_htmx.html

```diff
@@ -4,22 +4,34 @@
 <div class="row mb-3">
   <div class="col-auto d-print-none">
     <div class="input-group input-group-flat me-2 quicksearch" hx-disinherit="hx-select hx-swap">
-      <input type="search" results="5" name="q" id="quicksearch" class="form-control px-2 py-1" placeholder="Quick search"
-        hx-get="{{ request.full_path }}" hx-target="#object_list" hx-trigger="keyup changed delay:500ms, search" />
+      <input type="search" results="5" name="q" id="quicksearch" class="form-control px-2 py-1"
```

Comment:
We need to figure out how to make the new dropdown visually consistent with the quick search box.

![screenshot](https://github.com/netbox-community/netbox/assets/13487278/bfe78acb-f17f-4f6c-ac88-3faeafafdbb4)